CodeReviewChallenge.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+​
+public class Class1 {
+    public final String SPECIAL_CHARACTER = "a";
+​
+    public List<String> method(String paragraph) {
+        List<String> shortestWords = new ArrayList<>();
```

Comment:
Since we are looking for shortest words in the sentences containing the special character it should have a better name to describe what it holds